R-packages/covidcast/_pkgdown.yml

```diff
@@ -8,3 +8,43 @@ home:
 
     - text: View the COVIDcast map
       href: https://covidcast.cmu.edu/
+
+reference:
+  - title: Fetch data
+    desc: Fetch data from the COVIDcast API
+  - contents:
+      - covidcast_signal
+      - covidcast_signals
+      - covidcast_meta
+  - title: Analyze and plot data
+    desc: Plot and summarize signals
```

Comment:
I might suggest: 
title: Plot and summarize data
desc: Plot and summarize COVIDcast signals